browser/blueprints_test.py

```diff
@@ -1,4 +1,4 @@
-"""Test for Caliban Blueprints"""
+"""Test for DeepCell Label Blueprints"""
```

Comment:
Now resolved. There are still 2 broad TODOs that were there before this PR, but all the TODOs about 500 errors appearing instead of 404/other error codes are resolved. The issue was the `except Exception` blocks within the routes, which caught `HTTPExceptions` and made them into generic exceptions with a 500 error code.